packages/amplify-app/src/index.js

```diff
@@ -112,7 +112,7 @@ async function installAmplifyCLI() {
 async function amplifyCLIVersionCheck() {
   try {
     const amplifyCLIVersionRaw = await callAmplify(['-v'], { inheritIO: false });
-    const amplifyCLIVersionMatch = amplifyCLIVersionRaw.match(/\d+\.\d+\.\d+(-[a-z]+\.[0-9]+)?/g);
+    const amplifyCLIVersionMatch = amplifyCLIVersionRaw.match(/^\d+\.\d+\.\d/g);
```

Comment:
The last `+` looks important to me unless I'm missing something

```suggestion
    const amplifyCLIVersionMatch = amplifyCLIVersionRaw.match(/^\d+\.\d+\.\d+/g);
```

Reply:
it is, good catch.